reference/tbp-api.yml

```diff
@@ -198,18 +199,13 @@ paths:
             examples: {}
         description: ''
     parameters: []
-  '/users/{userID}/state/{userAttributeID}':
+  '/v1.0/users/{userID}':
     parameters:
       - schema:
           type: string
         name: userID
         in: path
         required: true
-      - schema:
-          type: string
-        name: userAttributeID
-        in: path
-        required: true
     patch:
       summary: ''
       operationId: patch-bots-botID-users-userID-state-userAttributeId
```

Comment:
could be `update-user-data`, just to be stylistically aligned with other operationIds we have in the API?